### ISSUE TITLE
TotalCross debugger viewer won't run on Windows 10

### ISSUE THREAD

**Issue by nmarquesin** [TotalCross/totalcross#153]
## Issue Title
TotalCross simulator won't run on Windows 10

## Describe the bug
When trying to run the simulator on the Hello World example with TotalCross V6.1.0, the application doesn't start and an exception is thrown:

![image](https://user-images.githubusercontent.com/23559837/94415706-d027c200-0175-11eb-9570-66d35197ee64.png)

In this case I am able to package and run the application on Windows.

I tried using the current branch master of V6.1.0. After editing pom.xml with 
```
<dependency>
            <groupId>net.harawata</groupId>
            <artifactId>appdirs</artifactId>
            <version>1.2.0</version>
</dependency>
```
the simulator did run, but I was unable to run the application on Windows. I got Fatal Error TCVM.dll not found even though I could see it in the folder.
 
![image](https://user-images.githubusercontent.com/23559837/94417242-c010e200-0177-11eb-8f54-7c29b8810403.png)

## To Reproduce
To reproduce the error, try running the simulator on a new TotalCross application.
Same as [this issue](https://github.com/TotalCross/embedded-samples/issues/17), it only happened with the Oracle JDK. It works fine with Open JDK

## Expected behavior
I can see the application on the simulator and can open it on my computer when packaging for win32.

## Devices:
 - Device: PC
 - OS: Windows
 - OS Version: 10
 - TotalCross Version: 6.1.0
 - Oracle JDK 11

**Comment by ricardobna** [comment]
Couldn't reproduce this bug, if you have any new infor on this issue feel free to update the issue, for now it's put on hold.

**Comment by VicMisael** [comment]
A VirtualBox virtual machine where the issue can be reproduced

https://drive.google.com/file/d/15kDipJljfzzoCOBQkLZ0wjUmnM7PteoA

**Comment by ricardobna** [comment]
The simulator bug was solved on #152 I'm currently working on the dll not found problem.

**Comment by ricardobna** [comment]
I tried a bunch of things using the provided vm environment and came to the conclusion that this problem is caused by the windows installation used, as it severely lacks essential installations, an example would be visual c++, it has nothing installed for that, what I suggest is to try this on a fuller version of windows vm and see if this still happens, for now this problem seems to be vm exclusive, in case you have this problem occur outside a vm environment, feel free to reopen this issue.
